packages/calypso-e2e/src/lib/pages/media-page.ts

```diff
@@ -43,6 +39,20 @@ export class MediaPage {
 		this.page = page;
 	}
 
+	/**
+	 * Waits until the Media page gallery is loaded and ready.
+	 */
+	async waitUntilLoaded(): Promise< ElementHandle > {
```

Comment:
Not 100% related to viewports, however media galleries can take some time to load.